aws-rds-dbcluster/src/main/java/software/amazon/rds/dbcluster/BaseHandlerStd.java

```diff
@@ -130,6 +152,15 @@ public abstract class BaseHandlerStd extends BaseHandler<CallbackContext> {
                     DbClusterRoleNotFoundException.class)
             .build();
 
+    protected static final ErrorRuleSet DESCRIBE_EVENTS_ERROR_RULE_SET = ErrorRuleSet
+            .extend(Commons.DEFAULT_ERROR_RULE_SET)
+            .withErrorCodes(ErrorStatus.ignore(OperationStatus.IN_PROGRESS),
+                    ErrorCode.AccessDenied,
+                    ErrorCode.AccessDeniedException,
+                    ErrorCode.NotAuthorized,
+                    ErrorCode.UnauthorizedOperation)
+            .build();
+
```

Comment:
Do we need this one, given there is an identical error rule set in `Events`

Reply:
Done.